js/Utilities.js

```diff
@@ -1341,9 +1335,5 @@ var downloadButton = function(name){
 
 				var xml = (mxUtils.getXml(root));
 
-				new mxXmlRequest(builder_path + "/download.php", $.param({
-					name: "Insight Maker Diagram",
-					"format": "svg",
-					"data": xml
-				})).simulate(document, "_blank");
+				downloadFile("Insight Maker Diagram.svg", xml)
```

Comment:
I tried it in Google Chrome and Firefox without it being blocked. I can do some more tests

The semi-colon is resolved! :)